pkg/plugins/golang/v4/scaffolds/init.go

```diff
@@ -81,26 +81,33 @@ func (s *initScaffolder) Scaffold() error {
 		machinery.WithConfig(s.config),
 	)
 
-	bpFile := &hack.Boilerplate{
-		License: s.license,
-		Owner:   s.owner,
-	}
-	bpFile.Path = s.boilerplatePath
-	if err := scaffold.Execute(bpFile); err != nil {
-		return err
-	}
+	if s.license != "none" {
+		bpFile := &hack.Boilerplate{
+			License: s.license,
+			Owner:   s.owner,
+		}
+		bpFile.Path = s.boilerplatePath
+		if err := scaffold.Execute(bpFile); err != nil {
```

Comment:
well, that's a good question.  'none' is valid even if not exist in knownLicenses[f.License]. 
Let me explain it for you.
If we pass '--license=none', then it will skip this(line 90).
In `scaffold.Execute(bpFile)`, it will call `SetTemplateDefaults(`) and `Validate()`.  So if we set 'none', it will never use `knownLicenses[f.License]`. Does it make sense? @camilamacedo86